### DISCUSSION TITLE
error trying to install ruby on rails

### DISCUSSION THREAD

**Discussion by ghost** [ruby/rubygems#7820] [Q&A] [answered]
Hello, how are you? I'm trying to install Ruby on Rails but it keeps saying this: C:\Windows\system32>gem install rails
ERROR: Could not find a valid gem 'rails' (>= 0), here is why: Unable to download data from [https://rubygems.org/](https://rubygems.org/) - IO::TimeoutError: Failed to open TCP connection to rubygems.org:443 (Blocking operation timed out!) ([https://rubygems.org/specs.4.8.gz](https://rubygems.org/specs.4.8.gz))
I have already tried to temporarily disable the firewall, but without success. The same error persists in any command, such as gem sources -a [http://rubygems.org/](http://rubygems.org)

**Comment by MSP-Greg** [comment, marked as answer]
There's a file located at https://github.com/rubygems/ruby-ssl-check/blob/master/check.rb, you can try and run that.

Depending on your OS version, a simpler way is to run the following from a command prompt:
```
curl -Lks 'https://git.io/rg-ssl' | ruby
```

Hopefully that will give you an idea why Ruby cannot make a connection.

Also, it's kind of odd to run commands from `C:\Windows\system32`...
